src/Session/commands.py

```diff
@@ -78,10 +78,31 @@ def list_freezers(self):
 
     def load(self, profile):
         pass
-    
-    def open(self, food_storage):
-        pass
+   
+    ''' Prompts user input to open a food storage. If a food storage with that name does not exist, 
+        prompts user to create a new one. '''
+    def open(self, foodStorage_type):
+        foodStorage_list = self.profile.listFoodStorages(foodStorage_type)
+        print(OPEN_FOOD_STORAGE_MESSAGE.format(foodStorage_list))
+        
+        user_foodStorage_type = input(OPEN_FOOD_STORAGE_NAME)
```

Comment:
I think the name here is misleading and should be something like `foodStorage_name`